packages/manager/modules/pci/src/projects/project/training/dashboard/install/install.controller.js

```diff
@@ -5,7 +5,9 @@ export default class PciTrainingInstallController {
     this.guideUrl = GUIDE_URL;
     [this.currentRegion] = this.regions;
     this.osInstall = {
-      'Linux / Mac OS': `curl ${this.currentRegion.cliInstallUrl} | sh`,
+      Linux: `${this.currentRegion.cliInstallUrl}/ovhai-linux.zip`,
+      'Mac OS': `${this.currentRegion.cliInstallUrl}/ovhai-darwin.zip`,
```

Comment:
Not mandatory but I think it's written in this way now.
```suggestion
      'macOS': `${this.currentRegion.cliInstallUrl}/ovhai-darwin.zip`,
```